containers/kubernetes/reference-content/lb-ingress-controller.mdx

```diff
@@ -1,237 +1,129 @@
+  </Message>
 
-Create the ingress object with this YAML manifest. Note that we use our DNS wildcard on the host stanza of this YAML file.
-
-```yaml
-apiVersion: networking.k8s.io/v1
-kind: Ingress
-metadata:
-  name: cafe-ingress
-spec:
-  rules:
-    - host: test.49087273-8296-46cc-a82c-f08cb9623ce2.nodes.k8s.fr-par.scw.cloud
-      http:
-        paths:
-          - path: /tea
-            pathType: Prefix
-            backend:
-              service:
-                name: tea-svc
-                port:
-                  number: 80
-          - path: /coffee
-            pathType: Prefix
-            backend:
-              service:
-                name: coffee-svc
-                port:
-                  number: 80
-```
+3. Apply the configuration:
 
-```bash
-# kubectl create -f cafe-ingress.yaml
-# kubectl get ing
-NAME           CLASS    HOSTS                                                                  ADDRESS   PORTS   AGE
-cafe-ingress   <none>   test.49087273-8296-46cc-a82c-f08cb9623ce2.nodes.k8s.fr-par.scw.cloud             80      4m11s
-```
+  ```bash
+  kubectl create -f coffee-ingress.yaml
+  ```
 
-You can test that this ingress is configured correctly by accessing your test application:
+4. Test the ingress:
 
-```bash
-# curl http://test.49087273-8296-46cc-a82c-f08cb9623ce2.nodes.k8s.fr-par.scw.cloud/coffee
-Server address: 100.64.0.181:8080
-Server name: coffee-5f56ff9788-68xs2
-Date: 28/Apr/2020:13:34:26 +0000
-URI: /coffee
-Request ID: 9d2ee64655b936384a64cf89e7a975b0
-```
+  ```bash
+  curl http://YOUR_SCALEWAY_DNS_WILDCARD/coffee
+  ```
 
-## Using a reserved IP as the IP address of your LoadBalancer
+## Using a Reserved IP with a LoadBalancer
 
-When creating Kubernetes services, you have the possibility to create a service of the `LoadBalancer` type.  
-This is what we have done above to expose the ingress controller.  
-That means that a call to the [cloud-controller-manager](https://github.com/scaleway/scaleway-cloud-controller-manager) is made. Meaning that it will spawn a new Load Balancer on Scaleway.  
-The "normal" behavior, in this case, is that a new public IP will be associated with this LoadBalancer. This means that if you want to create a DNS record with that IP, for instance, you have to do it every time you spawn a new Load Balancer. 
-There is a way to avoid that and to:
 
-- Reserve an IP address.
-- Put DNS records on it.
-- Re-use this IP address as many times as you want for different services (one service at a time).
+1. Reserve an IP address through the Scaleway API:
 
-### Reserving a LoadBalancer IP using the Scaleway API
+  ```bash
+  curl -X POST "https://api.scaleway.com/lb/v1/regions/$SCW_DEFAULT_REGION/ips" -H "X-Auth-Token: $SCW_SECRET_KEY" -H "Content-Type: application/json" \
```

Comment:
This is the wrong endpoint now - maybe you could just link to the anchor in the relevant section of this page: containers/kubernetes/reference-content/managing-load-balancer-ips.mdx